### PR TITLE
Organizations as read-only view, issns in concepts

### PR DESCRIPTION
**Pull request by daisieh**
The organization table doesn’t need to be an actual table in the database: it can be a read-only view from the Concepts. This PR creates that model and updates the related Java models to correspond to that. It also adds a hook into the view for the ISSNs.

I also upgraded us to Jackson 2 as it is a seamless upgrade.

### CODE REVIEW

**Comment by daisieh** [comment]
To test, make sure to run `psql < dspace/etc/postgres/dryad-rest-webapp.sql` before redeploying. Then do anything through either the metadata email system or the REST API and see that things still work properly. There shouldn't be any new visible functionality.

**Comment by ryscher** [comment]
Looks good.

File: dspace/etc/postgres/dryad-rest-webapp.sql [hunk 1]
```diff
@@ -50,3 +39,34 @@ CREATE TABLE rest_resource_authz
...
+  INNER JOIN metadatafieldregistry jid on journalschema.short_id='journal' and jid.metadata_schema_id=journalschema.metadata_schema_id and jid.element = 'issn'
+  WHERE jid.metadata_field_id = journal_id.field_id
+  ;
+
+DROP TABLE if EXISTS organization;
+-- can't have a foreign key constraint based on a view
+ALTER TABLE manuscript DROP CONSTRAINT if EXISTS manuscript_organization_id_fkey;
+
+CREATE VIEW organization AS
+SELECT journal_code_view.organization_id, code, fullname, issn
```
Comment (ryscher) [inline dspace/etc/postgres/dryad-rest-webapp.sql]: "fullname" should be "name"

**Comment by ryscher** [comment]
Tests good.

**Pull request merged**